packages/core/src/config.ts

```diff
@@ -1020,7 +1041,7 @@ const composeTargetConfig = (
 const composeExternalHelpersConfig = (
   externalHelpers: boolean,
   pkgJson?: PkgJson,
-): RsbuildConfig => {
+) => {
```

Comment:
why isolated declaration not throw an error

Reply:
Cause it's not exported?